packages/interfaces/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- (MINOR): Added an `audit` method to `IAppliance`.
```

Comment:
I… _think_ this should go under `### Added`? 🤔

Reply:
What makes you think that.

Reply:
Context clues.